lib/streamlit/__init__.py

```diff
@@ -76,6 +76,8 @@
 from streamlit.report_thread import add_report_ctx as _add_report_ctx
 from streamlit.report_thread import get_report_ctx as _get_report_ctx
 from streamlit.script_runner import StopException
+from streamlit.script_runner import RerunException as _RerunException
```

Comment:
@tconkling did you want `RerunException` to be a part of the API in the same way that `StopException` is?